config/textColors.json

```diff
@@ -0,0 +1,170 @@
+{
+	"colors": {
```

Comment:
1) Perhaps unify this code with `InterfaceObjectConfigurable::readColor`? E.g. add this new method in render/Colors.h that will load ColorRGBA from string

2) Perhaps reduce this list a bit? I definitely see some duplicated and clearly incorrect colors. Having 10-20 definitely working colors is better than 100 colors that might work

Reply:
Will have a look into it. :)

Reply:
Changed the list. Yes, as much as before. But: Now exactly the color names from HTML standard. You can look for the HTML color names now.

Reply:
Not want to change readColor atm.

Reply:
H3Colors are simply base colors with exception of H3 Pink, but you have to be a butterfly to distinguish those 2 apart.
```
Color code "F80000" has duplicates:
"r"
"H3Red"
"Red"

Color code "00FCF8" has duplicates:
"c"
"H3Cyan"
"Cyan"

Color code "00FC00" has duplicates:
"g"
"H3Green"
"Green"

Color code "0000F8" has duplicates:
"b"
"H3Blue"
"Blue"

Color code "F8FC00" has duplicates:
"y"
"H3Yellow"
"Yellow"

Color code "F88000" has duplicates:
"o"
"H3Orange"
"Orange"

Color code "F800F8" has duplicates:
"p"
"H3Purple"
"Purple"
```

Reply:
Yes, it's an alias. It makes clear which colors used originally. If you want to select "true" Heroes III colors.